src/Application/Clans/Models/ClanMemberViewModel.cs

```diff
@@ -8,4 +8,5 @@ public record ClanMemberViewModel : IMapFrom<ClanMember>
 {
     public UserPublicViewModel User { get; init; } = default!;
     public ClanMemberRole Role { get; init; }
+    public ClanPublicViewModel? Clan { get; set; }

```

Comment:
Nah the ClanMemberViewModel class is used to get all the members of a clan, we don't want to repeat the clan inside. I think you can just put a ClanPublicViewModel in UserPublicViewModel. It will miss the role but it's good enough for now.

Reply:
I though i needed to have the public view follow the same nesting as the entities. Is this better now?